src/UIState/SeeGooglemins.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * SeeGoogleMins.cpp
+ */
+
+#include "SeeGoogleMins.h"
+
+#include "../Devices/EEPROM_TC.h"
+#include "../Devices/LiquidCrystal_TC.h"
+
+void SeeGoogleMins::loop() {
+  char buffer[17];
+  sprintf(buffer, "%i", EEPROM_TC::instance()->getGoogleSheetInterval() / 60000);
```

Comment:
Are we storing minutes or milliseconds? What happens in `SetGoogleSheetInterval.cpp`?

Reply:
Looking at 0.197 we collect the value in minutes.